mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -747,6 +766,15 @@ func expandNodePool(d *schema.ResourceData, prefix string) (*container.NodePool,
 		}
 	}
 
+ <% unless version == 'ga' -%>
+	if v, ok := d.GetOk(prefix + "placement_policy"); ok {
+		placement_policy := v.([]interface{})[0].(map[string]interface{})
+		np.PlacementPolicy = &container.PlacementPolicy{
+			Type: placement_policy["type"].(string),
```

Comment:
I would also check if "type" is provided

Reply:
I think it's not necessary here, since I marked "placement_policy.type" with "Required: true" (placement_policy itself is optional). In some of the next lines it seems necessary (translation is the other way), added this check.

Reply:
Makes sense I wan unaware of this, looks good then

Reply:
One more change - apparently the check in the other place also doesn't work since PlacementPolicy.Type is a string and can't be null. It was not the PlacementPolicy from GKE API, but some generated code. Removed that check as well.